module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,15 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+
+std::vector<std::string> removeVowels(std::vector<std::string>& vec) {
+    std::vector<char> vowelVec{'a', 'e', 'i', 'o', 'u', 'y'};
```

Comment:
In my opinion you should use std::array with constexpr for example
```suggestion
    constexpr int arraySize = 12
    constexpr std::array<char, arraySize> vowelArray{'a', 'e', 'i', 'o', 'u', 'y','A', 'E', 'I', 'O', 'U', 'Y'};
```